lib_eio/net.mli

```diff
@@ -12,6 +12,8 @@
 *)
 
 exception Connection_reset of exn
+exception Unix_error of string * string * string
+exception Connection_failure of (string * string)
```

Comment:
No need for an extra tuple here.
```suggestion
exception Connection_failure of string * string
```
But what are these strings, anyway?

Reply:
`(host, port)`

Reply:
The exception is `Connection_failure of exn` now.